src/analysis/calculus/deriv_analytic.lean

```diff
@@ -0,0 +1,36 @@
+/-
```

Comment:
Could you maybe move this also to `fderiv_analytic`? I am not sure it makes sense to have two separate files here.

Reply:
Sure!

Reply:
Done.